src/Core/src/Core/IBoxView.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Microsoft.Maui
+{
+	/// <summary>
+	/// BoxView renders a simple rectangle of a specified width, height, and color.
+	/// </summary>
+	public interface IBoxView : IShapeView
```

Comment:
Do we need this? Is see the NeedsContainer checks for background, maybe the shapeview needs to check for fill or something?

Reply:
This seems really unnecessary; why is BoxView the only shape that has this issue? If it _is_ the only shape with this problem, can't the handler just check for the concrete BoxView type (rather than introducing a marker interface)?

Reply:
We need a container to draw the background in Shapes but not in the BoxView. In the BoxView the properties Color, BackgrounColor etc just fill the BoxView (RoundedRectangle). In the Core project, we don't have access to the concrete types.